### PR TITLE
Basic tests

### PR DESCRIPTION
**Pull request by Willo0ow**
- add vitest, happy-dom, vie-test-utils to the project
- create tests for useFormatMovieLength and validationRules helpers
- create tests form MovieCard

### CODE REVIEW

File: src/tests/unit/helpers/useFormatMovieLength.spec.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { formatMovieLength } from "@/helpers/useFormatMovieLength";
+import { describe, it, expect } from "vitest";
+
+describe("formatMovieLength helper", () => {
+  it("Formats movie length in number of minutes into a string with hours and minutes", () => {
+    expect(formatMovieLength(100)).toBe("1h 40min");
+  });
```
Comment (szymonlicau) [inline src/tests/unit/helpers/useFormatMovieLength.spec.js:7]: I'd only add a second case here where the number of minutes would be 0 (so a multiple of `60`) - because that case it is not known from this spec if it would be something like `1h` or `2h` or `1h 0min` 😄

**Review by szymonlicau — COMMENTED** [review]
I think this is a good start and a solid few tests - also good test candidates 😄

I left one suggestion which could leave some more clarity in the spec itself on how one of the functions work, other than that - good work! 😄

File: src/tests/unit/helpers/useFormatMovieLength.spec.js (hunk 1 above)
Comment (Willo0ow) [reply src/tests/unit/helpers/useFormatMovieLength.spec.js:7]: Yeah, you got me, that's actually an edge case I haven't considered while writing the function 😅
I'm on it!

**Review by szymonlicau — APPROVED** [review]
